target/linux/ramips/dts/mt7628an_iptime.dtsi

```diff
@@ -0,0 +1,109 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include "mt7628an.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	aliases {
+		label-mac-device = &ethernet;
```

Comment:
You are adding this for both devices a3 and a604m here, which is good and at the correct place (if you tested it).

However, since this changes stuff for a604m in addition to adding a3, I would rather make it a second commit though.

Reply:
You mean adding a3 label address in another commit?

Reply:
Since it is easier to do, just have your commit as it was before and then add the aliases block in a separate commit.

Reply:
Okay, done.

Reply:
Oh sorry, I'm stupid.
I just was overlooking the fact that a604m already had the label-mac-address set (I thought you were also adding it there).
Finally facing reality, I wouldn't have recommended introducing the separate commit in the first place. Anyway, it's also okay to have it.

So, you can now choose whether you want to keep the two-commit-approach or whether you want to go back again to your one-commit-solution, squashing the two.

Again, I'm sorry for the mess, but I just did not look carefully enough.

Reply:
No problem xD I'll revert then.

Reply:
Fine!